fe1-web/components/eventList/LaoProperties.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import {
+  TouchableOpacity, View, ViewStyle,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import { Spacing } from 'styles';
+import TextBlock from 'components/TextBlock';
+import styleEventView from 'styles/stylesheets/eventView';
+import ListCollapsibleIcon from 'components/eventList/ListCollapsibleIcon';
+import { connect } from 'react-redux';
+import ParagraphBlock from 'components/ParagraphBlock';
+import { Lao } from 'model/objects';
+
+function buildLaoProperties(lao: Lao) {
```

Comment:
for readability, I would suggest renaming as `renderLaoProperties`
but then I wonder, do we need a separate function for this?

Reply:
I separated this function from the main body in prevision for the "organizer LaoProperties". We will probably have something along the lines of `renderStandardProperties` and `renderOrganizerProperties`